### ISSUE TITLE
Datetime picker doesn't pop up anymore in WordPress version 5.3.2

### ISSUE THREAD

**Issue by LucianIacob** [carmoreira/date-time-picker-field#3]
After upgrade to wordpress 5.3.2 the date picker dialog doesn't open anymore, leads to user can insert anything in the date input field. 

Any thoughts/workarounds? Thank you.

**Comment by LucianIacob** [comment]
I just noticed that after i try to submit my form with errors, after error message comes like "all input fields needs to be completed", when i click on date input fields, **the pop-up is shown**. But only after i submit a form with errors first.

**Comment by carmoreira** [comment]
Hi, sorry for not having followed up before. Did the problem persist? Usually this means there's an external js error not allowing the picker to be triggered. Since this ticket is already a bit old, I'll close it, but if the problem persists or anyone has a similar issue, feel free to open a new ticket. Greetings, Carlos
